packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.Props.ts

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { ExpandingCard } from './ExpandingCard';
+import { IRenderFunction } from '../../Utilities';
+import { IStyle, ITheme } from '../../Styling';
+
+export interface IExpandingCard {
+
+}
+
+/**
+ * ExpandingCard component props.
+ */
+export interface IExpandingCardProps extends React.HTMLAttributes<HTMLDivElement | ExpandingCard> {
+  /**
+   * Optional callback to access the IExpandingCard interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IExpandingCard) => void;
+
+  /**
+   *  Item to be returned with onRender functions
+   */
+  renderData?: any;
+
+  /**
+   *  Render function to populate compact content area
+   */
+  onRenderCompactCard: IRenderFunction<IExpandingCardProps>;
+
+  /**
+   *  Render function to populate expanded content area
+   */
+  onRenderExpandedCard?: IRenderFunction<IExpandingCardProps>;
+
+  /**
+   * Element to anchor the ExpandingCard to.
+   */
+  targetElement?: HTMLElement;
+
+  /**
+   * Callback when the ExpandingCard tries to open
```

Comment:
actually...last idea here. since we currently have an enum for the card modes, how about simply have onModeChange, and pass in old/new mode. This will support more modes in the future without creating even more callbacks.